scaletest/templates/scaletest-runner/main.tf

```diff
@@ -625,7 +625,6 @@ resource "coder_agent" "main" {
     vscode     = false
     ssh_helper = false
   }
-  startup_script_timeout  = 86400
   shutdown_script_timeout = 7200
```

Comment:
```suggestion
```
Should we also remove `shutdown_script_timeout`?

Reply:
Yeah, that's also marked as deprecated. 👍